src/content/docs/en/community-resources/content.mdx

```diff
@@ -77,9 +78,15 @@ Have you written a recipe for Astro? [Edit this page](https://github.com/withast
 - [Add searching to your site with Pagefind](https://blog.otterlord.dev/posts/astro-search/)
 - [Add searching to your site with Fuse.js](https://www.youtube.com/watch?v=XnV_2MWqAhQ)
 - [Add a comments section to your Astro blog using Giscus](https://elazizi.com/posts/add-comments-section-to-your-astro-blog/)
+- [Using GreenSock Animation Platform (GSAP) in Astro](https://www.launchfa.st/blog/gsap-astro)
```

Comment:
I actually thought we had a category for ## Animation here. I think people are super interested in that, so let's make a new category now and include these here!